march_hardware/src/IMotionCube.cpp

```diff
@@ -69,6 +69,8 @@ void IMotionCube::mapMisoPDOs()
   map_miso.addObject(IMCObjectName::DetailedErrorRegister);
   map_miso.addObject(IMCObjectName::DCLinkVoltage);
   map_miso.addObject(IMCObjectName::MotorPosition);
+  map_miso.addObject(IMCObjectName::MotorVelocity);
+  map_miso.addObject(IMCObjectName::ActualVelocity);
```

Comment:
Why not call these IncrementalVelocity and AbosluteVelocity?

Reply:
To have consistency with the absolute and incremental encoders. I can adjust the names, but I suggest we also change the ones of the positions.